Source/WebCore/html/HTMLTemplateElement.cpp

```diff
@@ -155,10 +156,10 @@ void HTMLTemplateElement::attachAsDeclarativeShadowRootIfNeeded(Element& host)
         return;
 
     auto delegatesFocus = hasAttributeWithoutSynchronization(HTMLNames::shadowrootdelegatesfocusAttr);
-
     auto clonable = hasAttributeWithoutSynchronization(HTMLNames::shadowrootclonableAttr);
+    auto serializable = hasAttributeWithoutSynchronization(HTMLNames::shadowrootserializableAttr);
 
-    auto exceptionOrShadowRoot = host.attachDeclarativeShadow(*mode, delegatesFocus, clonable);
+    auto exceptionOrShadowRoot = host.attachDeclarativeShadow({ *mode, delegatesFocus, clonable, serializable });
```

Comment:
I don't think this will address my concern that the orders of delegatesFocus, clonable, & serializable can change without any type failures.
We should define 3 enum classes like these:
```
enum class ShadowRootDelegatesFocus : bool { No, Yes };
enum class ShadowRootIsClonable : bool { No, Yes };
enum class ShadowRootIsSerializable : bool { No, Yes }
```